base/src/dispatcher/dispatcher.ts

```diff
@@ -344,16 +344,16 @@ export class Dispatcher implements ZLUX.Dispatcher {
           return new RecognizerOr(...args);
         }
         else {
-          throw new Error(`No args provided for AND/OR recognizer op`);
+          throw new Error(`ZWED5020E - No args provided for AND/OR recognizer op`);
```

Comment:
Is this the correct way to format an error?  Shouldn't this be just a message ID?  But it appears you have done this for all error message calls so it seems there is something I am misunderstanding about the logger format.

Reply:
There is no mechanism to retrieve a message from an ID when logging a message. When that is built, we can redo this exercise and replace the hard-coded strings with the code.